src/components/TokenIcon.tsx

```diff
@@ -50,10 +60,21 @@ export default function TokenIcon({ token, viewStyle, testID, size = IconSize.ME
           testID={testID ? `${testID}/TokenIcon` : 'TokenIcon'}
         />
       ) : (
-        <DefaultToken
+        <View
+          style={[
+            styles.tokenCircle,
+            {
+              width: tokenImageSize,
+              height: tokenImageSize,
+              borderRadius: tokenImageSize / 2,
+            },
+          ]}
           testID={testID ? `${testID}/DefaultTokenIcon` : 'DefaultTokenIcon'}
-          size={tokenImageSize}
-        />
+        >
+          <Text style={[styles.tokenText, { fontSize: tokenTextSize }]}>
+            {token.symbol.substring(0, 4)}
```

Comment:
i don't think we want this text to scale, right?
```suggestion
          <Text style={[styles.tokenText, { fontSize: tokenTextSize }]} allowFontScaling={false}>
```